design/one-pager-strongly-typed-class.md

```diff
@@ -186,7 +186,7 @@ specTemplate:
     - sg-ab1cdefg
     - sg-05adsfkaj1ksdjak
   size: 20
-  providerRef:
+  providerConfigRef:
```

Comment:
Looks like there are still a number of classes where these changes don't need to be made

Reply:
Now it should be OK.